### DISCUSSION TITLE
Will --self-contained be removed any time soon?

### DISCUSSION THREAD

**Discussion by airtower-luna** [jgm/pandoc#9312] [General]
I noticed the deprecation warning for the `--self-contained` option when building documentation. It's easy enough to replace, but some relevant distro versions (namely Debian Bookworm, current stable) don't have Pandoc versions that support `--embed-resources` yet.

Is there a plan to *remove* `--self-contained` any time soon? #7331 mentions the general intention, but no schedule. Can I safely delay replacing it until relevant distros catch up (probably until Debian Trixie release)?

**Comment by jgm** [comment]
What's the timeline for Debian Trixie?

**Reply by airtower-luna** [reply]
Bookworm was released last summer, and there are about 2 years between Debian releases. If that holds the Trixie release should be in summer 2025, the [freeze policy](https://release.debian.org/trixie/freeze_policy.html) still says "TBA".

**Reply by jgm** [reply]
Well, I don't have any plans to remove the option soon.  (I can't guarantee I'll remember, but I'll keep this in mind.)

**Reply by airtower-luna** [reply]
Thank you! Then I'll shelve the change for now, ideally until Trixie release.
